.gitignore

```diff
@@ -43,6 +43,11 @@ pip-log.txt
 nosetests.xml
 
 # generated examples
-/examples/boards/ppro/xilinx
-/examples/build/xilinx
+/examples/*/xilinx
+/examples/*/*/xilinx
```

Comment:
You shouldn't need /*/* a single /*/ should cover it, no?  (simple tests verify only /*/ is needed).